app/portainer/interceptors/redirectStatusInterceptor.js

```diff
@@ -0,0 +1,24 @@
+angular.module('portainer.app').factory('RedirectStatusInterceptor', [
+  '$q',
+  '$state',
+  function ($q, $state) {
+    'use strict';
+    var interceptor = {};
+
+    interceptor.responseError = responseErrorInterceptor;
+
+    function responseErrorInterceptor(rejection) {
+      if (rejection.status === 307 || rejection.status === 308) {
+        var redirectReason = rejection.headers()['redirect_reason'];
```

Comment:
need to check if rejection.headers is defined
```suggestion
	const headers = rejection.headers()
        const redirectReason = headers && headers['redirect_reason'];
```

Reply:
fixed